src/components/CreateResolution/CompleteResolution.vue

```diff
@@ -653,6 +662,21 @@ export default class CompleteResolution extends Mixins(CommonMixin, DateMixin, E
     this.updateResolutionStepValidationDetail()
   }
 
+  private async reRenderResolutionText (): Promise<void> {
+    this.$refs.resolutionTextRef.calculateInputHeight()
+  }
```

Comment:
This is such a small method, and only called from 1 place. Could its body just be moved to the caller?

(And same on CompleteResolutionSummary page.)

Reply:
yes

Reply:
Removed both instances of `reRenderResolutionText` function and moved logic into `onResolutionVisibilityChanged` function.